helm/provisioner/values.yaml

```diff
@@ -76,6 +76,8 @@ classes:
   # Uncomment to create storage class object and configure it.
   # storageClass:
     # reclaimPolicy: Delete # Available reclaim policies: Delete/Retain, defaults: Delete.
+    # isDefaultClass: true # set as default class
```

Comment:
Ah I forgot, can you also update the helm/README.md with the new option as well?

Reply:
done, but tests failed and do not look like linked to my changes